src/codacy_checkov.py

```diff
@@ -53,11 +53,16 @@ def runCheckov(config):
     file_opts = ([opt for f in config.files for opt in ['-f', f]]
                  if config.files
                  else ['-d', '.'])
+    processEnv = os.environ.copy()
+    processEnv["http_proxy"] = "http://foo"
+    processEnv["https_proxy"] = "https://foo"
```

Comment:
I am fine with them here instead of Dockerfile, if we prefer, but then can we move the 2 that are there to this format? https://github.com/codacy/codacy-checkov/blob/master/Dockerfile#L10 It would be cool that we have those all in the same place